Android/app/src/main/java/com/example/kahye/common/PostTest.java

```diff
@@ -0,0 +1,68 @@
+package com.example.kahye.common;
+
+import org.json.simple.JSONObject;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.Body;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class PostTest {
+    public interface CommonService {
+        @Headers("Content-Type: application/json")
+        @POST("api/login/")
+        Call<User> sendPost(@Body JSONObject userEmail);
+    }
+
+    class User {
+        @Expose
+        public String userEmail;
+        @SerializedName("userName")
+        @Expose
+        public String userName;
+        public String accountType;
+        public boolean isLecturer;
+    }
```

Comment:
I wrote this on purpose because It is just an example to make it easier to understand. 
I'll use the model Woongjin made.